src/surfaces/drawer/drawerDataGrid.d.ts

```diff
@@ -0,0 +1,41 @@
+import { number, string } from 'prop-types';
+import React from 'react';
+
+export interface DrawerDataGridPropTypes {
+    bleed?: boolean;
+    classes?: {
+        bleed?: string;
+        root?: string;
+    };
+    className?: string;
+    columns: object[];
+    data: object[] | {};
```

Comment:
NAB: Should probably be `| object` for consistency (favoring `object` over `{}`)

Reply:
Fixed.

Reply:
https://github.com/saddlebackdev/react-cm-ui/pull/346/commits/373dffa65bef7979afbf129396dec1f44990425d#diff-bc22e36464368ffeb1ec566250800215d81ffd2a50c10eb7384d2083baccbf91L12-R12
Looks good.  Resolving.